adafruit_pn532/adafruit_pn532.py

```diff
@@ -256,9 +269,13 @@ def _read_frame(self, length):
         # Return frame data.
         return response[offset + 2 : offset + 2 + frame_len]
 
-    def call_function(
-        self, command, response_length=0, params=[], timeout=1
-    ):  # pylint: disable=dangerous-default-value
+    def call_function(  # pylint: disable=dangerous-default-value
+        self,
+        command: int,
+        response_length: int = 0,
+        params: Optional[bytearray] = [],
+        timeout: float = 1,
+    ) -> bytearray:
```

Comment:
This can return `None` and can return `Union[bytes, bytearray]`, so the return should be `Optional[Union[bytes, bytearray]]`.